modules/nw-metallb-l2padvertisement-cr.adoc

```diff
@@ -0,0 +1,40 @@
+
+:_content-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-configure-address-pools.adoc
+[id="nw-metallb-l2padvertisement-cr_{context}"]
+= About the L2Advertisement custom resource
+
+The fields for the `l2Advertisements` object are defined in the following table:
+
+.L2 advertisements configuration
+[cols="1,1,3a", options="header"]
+|===
+
+|Field
+|Type
+|Description
+
+|`metadata.name`
+|`string`
+|Specifies the name for the L2 advertisement.
+
+|`metadata.namespace`
+|`string`
+|Specifies the namespace for the L2 advertisement.
+Specify the same namespace that the MetalLB Operator uses.
+
+|`spec.ipAddressPools`
+|`string`
+|Optional: The list of IPAddressPools to advertise with this advertisement, selected by name.
+
+|`spec.ipAddressPoolSelectors`
+|`string`
+|Optional: A selector for the IPAddressPools that gets advertised with this advertisement. If no IPAddressPool is selected by this or by the list, the advertisement is applied to all the IPAddressPools.
+
+|`spec.nodeSelectors`
```

Comment:
If we document this field, we need to declare it as tech preview (same for bgpadvertisement)